app/assets/stylesheets/styles/_header.scss

```diff
@@ -0,0 +1,7 @@
+nav {
+  background-color: #333;
```

Comment:
Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.